plugins/colorbutton/plugin.js

```diff
@@ -370,6 +414,152 @@ CKEDITOR.plugins.add( 'colorbutton', {
 			}
 		}
 
+		function renderContentColors( options ) {
+			// This function is called on each dialog opening.
+			var contentColorsRow = options.contentColorsRow,
+				contentColorsLabel = options.contentColorsLabel,
+				cssAttribute = options.cssAttribute,
+				clickFn = options.clickFn,
+				type = options.type,
```

Comment:
Overall destructuring properties to avoid "talking to strangers" is good practice, however, there is nothing wrong with accesing children members by a single dot. Destructuring `options` here makes this function less readable because later in function code you have to find out from where these variables come from. It would be a bit easier to recognize them as `options` children which also gives some context about its purpose. Also, see https://en.wikipedia.org/wiki/Law_of_Demeter for some tips about the object's loose coupling.